linux/scripts/debian.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+# Build source packages suitable to import into the Debian package repos
+# (https://salsa.debian.org/input-method-team)
+
+# must be run from linux dir
+
+# parameters: [PROJECT="<project>"] ./scripts/debian.sh
+# PROJECT="<project>" only upload this project
+
+set -e
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/../../resources/build/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+# Check the tier
+if [[ -z "${TIER}" ]]; then
+    echo "TIER.md or \${TIER} must be set to (alpha, beta, stable) to use this script"
+    exit 1
+fi
+
+if [ ! $(which xmllint) ]; then
+    echo "you must install xmllint (libxml2-utils package) to use this script"
+    exit 1
+fi
+
+if [ "${PROJECT}" != "" ]; then
+    projects="${PROJECT}"
+else
+    projects="keyman-keyboardprocessor kmflcomp libkmfl ibus-kmfl keyman-config ibus-keyman"
+fi
+
+BASEDIR=$(pwd)
+
+rm -rf debianpackage
+mkdir -p debianpackage
+
+for proj in ${projects}; do
+    if [ "${proj}" == "keyman-keyboardprocessor" ]; then
+       cd ${BASEDIR}/../common/core/desktop
+    else
+       cd ${proj}
+    fi
+
+    if [ "${proj}" == "keyman-config" ]; then
+        make clean
+    fi
```

Comment:
1. Does this new file replace the existing `deb.sh` script?
2. The first half of this file is pretty similar to launchpad.sh. Would it be better to refactor some for maintainability? (up to you which is preferred)

Reply:
I think `deb.sh` builds a source + binary package locally (haven't tried it, and I'm not aware that it is mentioned in the docs anywhere).
`debian.sh` builds a source package only.

Good idea to refactor the common code. Done.